### DISCUSSION TITLE
Video size

### DISCUSSION THREAD

**Discussion by Antoine99-jpg** [aclap-dev/video-downloadhelper#839] [Help and Q&A]
### Your question

With the old version you could see the size of the videos to download and choose the right link? Now it's impossible, and you end up downloading ads. Can you display video size please?

### Addon & CoApp version and details

When you have several download options, you end up making mistakes.

**Comment by GrampaWildWilly** [comment]
This is a faulty problem description.  You need to actually read the instructions that you got when you were creating this & follow them.  Please try again.  This is not just me being pedantic & trying to force you to follow rules.  What you have posted is utterly useless.  Nobody can help you based on what you've posted.  There's no useful information there.  Post a proper problem description & you'll get some useful help.
